pages/docs/data-ops/cli/cache.mdx

```diff
@@ -1,16 +1,15 @@
 import { Callout, Steps } from "nextra-theme-docs";
 
-# Hashboard cache command
+# `cache`
```

Comment:
@glean-calder  heads up I actually think having "Hashboard cache command" as the heading is better for SEO

Reply:
Hmm...why exactly / do we know how much better?

Taking this to the fullest extreme, we could put `hashboard` in every title on our docs page 🤷

Reply:
I don't know exactly how much better.  If someone just lands on our caching page (through SEO or some other deep link), it is a bit confusing to just have the title be 'cache' and without much additional context.  Example of how other popular tools document their CLI's: https://docs.getdbt.com/reference/commands/compile

Why do we have to take it to an extreme, can we just keep these titles please?

Reply:
Good enough for me! Will change the titles back